content/docs/beyond-the-basics/verified-domain/index.md

```diff
@@ -0,0 +1,46 @@
+---
+ title: "Verified Domain"
+ description: "Domain Verification and Why It's Important"
+ lead: "Domain Verification and Why It's Important"
+ date: 2021-05-18T14:52:25+01:00
+ lastmod: 2021-05-18T14:52:25+01:00
+ draft: false
+ images: []
+ menu:
+   docs:
+     parent: "beyond-the-basics"
+ weight: 43
+ toc: true
+---
+
+## What is domain verification?
+
+Domain verification assures that actors claiming to be part of an organization are authorized to share information on their behalf. If an organization's tenancy has been verified by the RKVST team, a badge indicating that they have been verified will appear next to their domain name.
+
+{{< note >}}
+**Note:** Having a verified domain is different from a `Tenant Display Name`. Tenant display names are internal, appearing only within your own tenancy, and are not visible to anyone you share with. A verified domain name must be set by the RKVST team, and will be visible to actors outside your tenancy. 
+{{< /note >}}
+
+## Why is it important to verify my organization's domain?
+
+Getting your organization's domain verified indicates that you are who you say you are to those you wish to share with. This helps close the trust gap inherent to information sharing between organizations or with the public.
+
+Without domain verification, the `Organisation` is noted as the publisher's Tenant ID. Verifying your domain not only shows that this information comes from a legitimate actor on behalf of the organization, but also replaces the Tenant ID with your domain name so consumers can more easily identify the publishing organization. For example, someone attesting information on behalf of RKVST would have `rkvst.com`. 
+
+{{< img src="UnverifiedDomain.png" alt="Rectangle" caption="<em>Organisation without Verified Domain</em>" class="border-0" >}}
+
+{{< img src="DomainBadge.png" alt="Rectangle" caption="<em>Organisation with Verified Domain</em>" class="border-0" >}}
+
+## How can I get my organization's domain verified? 
+
+The RKVST team is happy to help you obtain your verified domain badge. Please contact support@rkvst.com from an email address which includes the domain you wish to verify. For example, email us from @rkvst.com to verify the rkvst.com domain. 
```

Comment:
Question: This made me wonder if we have writing / language guidelines as a team i.e. should we standardise on American English or British English throughout our work? Possibly a question for @JAG-UK ?

Reply:
The documentation is written in American English, according to the contributing guidelines that were previously set. There are a few places in the product (i.e. Organisation) that use British spellings, in which case I have kept direct references to those names spelled the same. 

Guidelines here: https://docs.rkvst.com/contributing/formatting-content/style-guide/#grammar